OCPBUGS-10304: Document oc-mirror command for creating ImageContentSou… (#63332)
openshift/openshift-docs: modules/oc-mirror-mirror-to-mirror.adoc

. Navigate into the `oc-mirror-workspace/` directory that was generated.
. Navigate into the results directory, for example, `results-1639608409/`.
. Verify that YAML files are present for the `ImageContentSourcePolicy` and `CatalogSource` resources.

[NOTE]
====

> The `repositoryDigestMirrors` section of `ImageContentSourcePolicy` is used for the `install-config.yaml` file during installation.

Reviewer: Need to clarify what the ImageContentSourcePolicy is here.
```suggestion
The `repositoryDigestMirrors` section of  the`ImageContentSourcePolicy` file/resource is used for the `install-config.yaml` file during installation.
```

Author: This sentence was removed by a different writer and no longer appears in the file.

Author: Done, thank you!

> Please note the `repositoryDigestMirrors` section of `ImageContentSourcePolicy` will be used for install-config.yaml file during installation.

Reviewer 2: ```suggestion
The `repositoryDigestMirrors` section of `ImageContentSourcePolicy` is used for the `install-config.yaml` file during installation.
```

Author: Done, thank you!
